### PR TITLE
Description field added + minor fixes

### PR DESCRIPTION
**Pull request by hudolejev**
See also: https://github.com/elasticsearch/logstash-contrib/pull/134

### CODE REVIEW

File: lib/logstash/outputs/jira.rb [hunk 1]
```diff
@@ -83,17 +83,22 @@ def receive(event)
...
+
 issue = Jiralicious::Issue.new
     issue.fields.set_id("project", @projectid) # would have prefered a project key, https://github.com/jstewart/jiralicious/issues/16
-    issue.fields.set("summary", @summary)
+    issue.fields.set("summary", summary)
+    issue.fields.set("description", event.sprintf(event.to_hash.to_yaml))
     issue.fields.set_id("issuetype", @issuetypeid)
-    issue.fields.set_name("reporter", @reporter)
-    issue.fields.set_name("assignee", @assignee)
+    issue.fields.set_name("reporter", @reporter) if not @reporter.nil?
```
Comment (wiibaa) [inline lib/logstash/outputs/jira.rb]: @hudolejev `if @reporter` is enough in ruby to check for non null value

File: lib/logstash/outputs/jira.rb [hunk 2]
```diff
@@ -83,17 +83,22 @@ def receive(event)
...
+    issue.fields.set_name("assignee", @assignee) if not @assignee.nil?
```
Comment (wiibaa) [inline lib/logstash/outputs/jira.rb]: same as above

File: lib/logstash/outputs/jira.rb [hunk 3]
```diff
@@ -83,17 +83,22 @@ def receive(event)
...
 
+    # Limit issue summary to 255 characters
+    summary = event.sprintf(@summary)
+    summary = "#{summary[0,252]}..." if summary.length > 255
+
 issue = Jiralicious::Issue.new
     issue.fields.set_id("project", @projectid) # would have prefered a project key, https://github.com/jstewart/jiralicious/issues/16
-    issue.fields.set("summary", @summary)
+    issue.fields.set("summary", summary)
+    issue.fields.set("description", event.sprintf(event.to_hash.to_yaml))
```
Comment (wiibaa) [inline lib/logstash/outputs/jira.rb:142]: @hudolejev can you add in the plugin documentation an example of output of this field please

File: lib/logstash/outputs/jira.rb (hunk 1 above)
Comment (hudolejev) [inline lib/logstash/outputs/jira.rb]: Fixed in ac83d02.

File: lib/logstash/outputs/jira.rb (hunk 3 above)
Comment (hudolejev) [inline lib/logstash/outputs/jira.rb:142]: Added: d98b330

**Comment by jsvd** [comment]
LGTM

**Pull request closed without merging**